### PR TITLE
Adds example app

### LINKED ISSUE #13: Add an example app
An example app would help show sample `react-csjs` usage and also assist with local development

### PR DESCRIPTION
**Pull request by tizmagik**
Closes #13

### CODE REVIEW

**Comment by tizmagik** [comment]
@wKovacs64 or @drcmda could one of you review please? Thanks!

File: example/src/client/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
```
Comment (wKovacs64) [inline example/src/client/index.js:3]: Need to add `react-hot-loader` as a dependency?

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "react-csjs-example",
+  "version": "1.0.0",
+  "description": "Example app showing simple react-csjs use",
+  "scripts": {
+    "dev": "kyt dev",
+    "test": "kyt test"
+  },
+  "author": "Jeremy Gayed (http://github.com/tizmagik)",
+  "license": "MIT",
+  "dependencies": {
```
Comment (wKovacs64) [inline example/package.json:11]: kyt docs recommend adding it as a dependency to your project, which we should probably do (I didn't have it installed, so `run dev` failed immediately).

File: example/src/client/index.js (hunk 1 above)
Comment (tizmagik) [reply example/src/client/index.js:3]: Technically it's provided by kyt, but yea let me add that to be more explicit

File: example/package.json (hunk 2 above)
Comment (tizmagik) [reply example/package.json:11]: Ah true yea sorry forgot to check that in

**Comment by tizmagik** [comment]
@wKovacs64 added those two deps -- nice catch!

**Review by wKovacs64 — APPROVED** [review]

**Pull request merged**